src/parcels/mount-parcel.js

```diff
@@ -46,6 +47,7 @@ export function mountParcel(config, customProps) {
     status: passedConfigLoadingFunction ? LOADING_SOURCE_CODE : NOT_BOOTSTRAPPED,
     customProps,
     owningAppOrParcel,
+    isParcel: true,
```

Comment:
Do we need this boolean? I think we could key off of `owningAppOrParcel` to determine if an object is a parcel or an application.

Reply:
👍 will remove